build.gradle

```diff
@@ -336,8 +336,8 @@ dependencyManagement {
 
 dependencies {
 
-    implementation group: 'com.github.hmcts', name: 'et-common', version: '2.3.0'
-    implementation group: 'com.github.hmcts', name: 'et-data-model', version: '3.17.1'
+    implementation group: 'com.github.hmcts', name: 'et-common', version: 'c8003b2b90'
```

Comment:
Will approve once https://github.com/hmcts/et-common/pull/588 has been merged and released with the tag updated here

Reply:
Created 2.3.1 version of et-common and updated et-common version.